plugins/woocommerce/changelog/40865-enhancement-enable-cart-and-checkout-blocks-as-the-default-checkout-experience-on-woocommerce

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Sounds like a major change?

Reply:
Do we need to have a way for hosts/providers to opt out of this for a period of time? cc @pmcpinto